indexing-service/src/main/java/io/druid/indexing/common/task/Task.java

```diff
@@ -47,6 +47,8 @@
     @JsonSubTypes.Type(name = "restore", value = RestoreTask.class),
     @JsonSubTypes.Type(name = "index", value = IndexTask.class),
     @JsonSubTypes.Type(name = "index_hadoop", value = HadoopIndexTask.class),
+    @JsonSubTypes.Type(name = "hadoop_convert_segment", value = HadoopConverterTask.class),
+    @JsonSubTypes.Type(name = "hadoop_convert_segment_sub", value = HadoopConverterTask.ConverterSubTask.class),
```

Comment:
Ok, I'm not deprecating them as part of this PR because that deserves its own discussion. And since this shares code with the Indexer Converter, it has to maintain compatibility until that one is also changed.